src/nvim/macros.h

```diff
@@ -154,6 +154,6 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
-#define RGB(r, g, b) ((r << 16) | (g << 8) | b)
+#define NVIM_RGB(r, g, b) ((r << 16) | (g << 8) | b)
```

Comment:
You could probably reduce the line count of this PR a bit by extracting this into a separate PR (and the syntax.c changes, of course).